dev-environment.md

```diff
@@ -43,8 +43,23 @@ list, and send an email. We will usually reply within 24 hours.
 Common Issues
 -------------
 
-One common issue is sugar-build failing for people behind **proxies**.  This is
-usually marked by an issue with the `volo` command, for example:
+One common issue is sugar-build failing for people behind **proxies**. This is
+usually because of pip proxy error or is marked by an issue with the `volo`
+command.
+
+Regarding pip issues, it raises the following error while installing python
+packages:
+    
+    MissingSchema: Proxy URLs must have explicit schemes.
+
+A workaround for this issue is that if you have an environment variable like
+`http_proxy=localhost:3128`, then explicitly add the URL schema by prefixing
+*http* and update it to `http_proxy=http://localhost:3128`. Similarly, update
+for the `http_proxy` environment variable.
```

Comment:
This issue is caused if you have no schema for your `http_proxy` environment variable.  "Explicitly add the URL schema by prefixing *http* , for example `http_proxy=localhost:3128` would be changed to `http_proxy=http://localhost:3128`."

Does that edit make more sense?